src/core/server/integration_tests/saved_objects/service/lib/repository.test.ts

```diff
@@ -46,8 +46,10 @@ describe('SavedObjectsRepository', () => {
   });
 
   afterAll(async () => {
-    await esServer.stop();
-    await root.shutdown();
+    if (root) {
```

Comment:
I wonder if we shouldn't be stopping root first. IMO it would make more sense (as it is done in the other test modified by this PR).

Reply:
We can but do we need to?